datadayessentials/data_transformation/_transformers.py

```diff
@@ -1009,3 +1013,50 @@ def process(self, df_in: pd.DataFrame):
                 df_in[col] = cat_series
         df_in = pd.concat([df_in, pd.DataFrame(num_dict)], axis=1)
         return df_in
+
+
+class DataFrameColumnTypeSplitter:
```

Comment:
The class should inherit from the interface IDataFrameTransformer.  With this format, the class needs to have a process method.  Then it can be used in a pipeline.  If you look at the other transformers in the class, they all follow the same structure.

You can then instantiate a DataFramePipe from _data_pipe.py and pass a list of transformers that will be run in sequence.

Reply:
updated inline with above